portfolio/src/main/webapp/index.html

```diff
@@ -6,14 +6,25 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body style = "background-color:rgb(194, 190, 190);"></body>
+      
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
+      <h1>Sofia Riley</h1>
+      <p style = "font-size: 30px;">Important Links:</p>
+      <a href ="https://www.linkedin.com/in/sofiariley/"> LinkedIn</a>
+      <a href ="https://github.com/SofiaPSU/sps-portfolio"> Github</a>
+      <p style ="font-size:30px">Introduction</p>
+      
+      <p>Hi Everyone! My name is Sofia and I am sophomore at Penn State!</p>
+      <p> I am currently majoring in Computer Science and planning to minor in Cyber Security.</p>
+      <p> On campus I am involved with the Assocation of Women in Computing, the Penn State Equestrian Team, the Girls Who Code College Loop, and the Women in Engineering Program.</p>
+      <p> I am a Technology Lead for the Girls Who Code College Loop and a Facilitator for the study groups organized by the Women in Engineering Program.</p>
```

Comment:
@GoogleyGabriel mentioned me that lines should be less than 100 characters, so keep that in mind; it appened to me too!